test/integration/multikueue/suite_test.go

```diff
@@ -222,27 +226,26 @@ func managerAndMultiKueueSetup(ctx context.Context, mgr manager.Manager, gcInter
 	gomega.Expect(err).NotTo(gomega.HaveOccurred())
 }
 
-func multiclusterSetup(gcInterval time.Duration) {
+var _ = ginkgo.BeforeSuite(func() {
 	var managerFeatureGates []string
 	version, err := versionutil.ParseGeneric(os.Getenv("ENVTEST_K8S_VERSION"))
 	if err != nil || !version.LessThan(versionutil.MustParseSemantic("1.30.0")) {
 		managerFeatureGates = []string{"JobManagedBy=true"}
 	}
 
-	managerTestCluster = createCluster(func(ctx context.Context, mgr manager.Manager) {
-		managerAndMultiKueueSetup(ctx, mgr, gcInterval)
-	}, managerFeatureGates...)
+	// pass nil setup since the manager for the manage cluster is different in some specs.
+	managerTestCluster = createCluster(nil, managerFeatureGates...)
```

Comment:
I don't think I follow, why the kueue manager setup is moved from here, but the kueue manager for workers stays here? I think it will be easier to understand if both types of manager where setup in the same place.

Reply:
There is no point in restarting the managers in the workers as long no configuration change is needed.

Reply:
I see, ok